chipy_org/apps/meetings/models.py

```diff
@@ -131,11 +140,37 @@ def is_future(self):
     def rsvp_user_yes(self):
         raise NotImplementedError
 
-    def rsvp_user_maybe(self):
-        raise NotImplementedError
-
     def number_rsvps(self):
-        return self.rsvp_set.exclude(response="N").count()
+        return self.rsvp_set.exclude(response=RSVP.NO).count()
+
+    def number_in_person_rsvps(self):
+        return self.rsvp_set.filter(
+            response=RSVP.YES, status=RSVP.ACCEPTED, venue=RSVP.IN_PERSON
+        ).count()
+
+    def number_virtual_rsvps(self):
+        return self.rsvp_set.filter(
+            response=RSVP.YES, status=RSVP.ACCEPTED, venue=RSVP.VIRTUAL
+        ).count()
```

Comment:
That you created helper methods based around [the concept I talk about here](https://github.com/chicagopython/chipy.org/pull/428/files#r754746545) is a sign that you should merge these values down. If your code base is scaling with pairs of database options then it makes sense to merge them in the database as well.

Reply:
I agree on merging the response and status. Would you just migrate all the old "YES" values to IN_PERSON?  Probably doesn't matter as we have never done analysis on old RSVP data.

Reply:
Yeah, I think the right answer here is to figure out what stats we care about from prior attendance and create a single record with that information and delete old RSVPs. Rationale: this is our largest row generator and we hit costs thresholds by row count.

But you could safely do everything pre 3/2021 as `attending_in_person` and everything since then as `attending_virtually`

Reply:
merged